jsx/Tabs.js

```diff
@@ -171,7 +171,56 @@ TabPane.propTypes = {
   activeTab: React.PropTypes.string
 };
 
+class Modal extends React.Component {
+
+  render() {
+    // Render nothing if the "show" prop is false
+    if(!this.props.show) {
+      return null;
+    }   
+
+    //White Modal Window
+    const modalStyle = { 
+      position: 'relative',
+      maxWidth: 1000,        
+      maxHeight: '100%',
+      margin: '0 auto',
+      backgroundColor: '#fff',
+      padding: 30,
+      borderRadius: 10,
+      overflowY: 'auto',
+      zIndex: 9999,
+    };  
+
+    //Grey Background
+    const backdropStyle = {
+      position: 'fixed',
+      zIndex: 9998,
+      top: 0,
+      bottom: 0,
+      left: 0,
+      right: 0,
+      backgroundColor: 'rgba(0,0,0,0.3)',
+      padding: 50
+    };
+
+    return (
+    <div style={backdropStyle} onClick={this.props.onClose}>
+      <div style={modalStyle} onClick={(e) => {e.stopPropagation()}}>
```

Comment:
This seems like it would be more consistent with the rest of the LORIS theme if it used a bootstrap modal dialog instead of trying to hack its own with inline CSS.

Reply:
I agree, but I was hoping that this react component would eventually replace the bootstrap modal dialogue in the future. Much like the accordion panel data-toggle, the use of bootstrap in React often makes it more difficult to implement custom functionality because we are constrained by the event cascades and formatting dictated by bootstrap. Although this particular modal window definitely needs more work, I see this as opportunity to develop a modal window that is catered to LORIS. Although, that logic applies a lot less for the modal window than it does for the panel data-toggle. @davidblader, any thoughts?